theatre/studio/package.json

```diff
@@ -28,6 +28,7 @@
   "dependencies": {
     "@theatre/dataverse": "workspace:*",
     "react": "^17.0.2",
+    "react-colorful": "^5.5.1",
```

Comment:
Should we bundle this instead? we're bundling most other packages already.

Reply:
You mean bundle with core? This is a studio-only library so I don't see a reason.

Reply:
Bundle with studio I mean

Reply:
Ah I understand. I think I'm gonna need help with configuring that. I see there's a bunch of stuff in theatre/package.json:devDependencies, is that something to do with this? The createBundle function defines externals, rather than packages to be bundled, so that should be fine I think, but my knowledge is lacking in this regard.

Reply:
Right. For now, we're just defining the dependencies on `theatre/package.json` unless we don't want those dependencies to get bundled, in which case we define them in `theatre/(core|studio)/package.json`.

- [ ] Let's add a quick note to document this

Reply:
Right, and what goes in dev deps and what in normal deps?

Reply:
Added notes about this.